src/styles/loops/loop2.scss

```diff
@@ -0,0 +1,7 @@
+@each $day, $number in $days {
```

Comment:
Would be better to move this loop inside the .calendar__cell selector. No need to create a separate file for this purpose